src/components/MenuItem.js

```diff
@@ -1,12 +1,23 @@
-import "../pages/Menu.css";
+import "./MenuItem.css";
+import useCount from "../hooks/useCount";
 
 export default function MenuItem({ item }) {
+  const [count, increment, decrement] = useCount();
   return (
     <div className="menu__item">
       <p>{item.type}</p>
       <p>{item.name}</p>
       <p>{item.description}</p>
-      <p>{item.price / 100}</p>
+      <div className="menu__div">
+        <p>{item.price / 100}</p>
+        <button className="menu__buttons" onClick={increment}>
+          +
+        </button>
+        {count}
```

Comment:
You could wrap the count into a span so you can style it independently (maybe making the font size bigger)